src/pybind/mgr/nfs/ganesha_conf.py

```diff
@@ -208,50 +209,52 @@ def to_dict(self) -> Dict[str, Any]:
 class CephFSFSAL(FSAL):
     def __init__(self,
                  name: str,
-                 user_id: Optional[str] = None,
+                 cmount_path: Optional[str] = None,
+                #  user_id: Optional[str] = None,
                  fs_name: Optional[str] = None,
-                 sec_label_xattr: Optional[str] = None,
-                 cephx_key: Optional[str] = None) -> None:
+                 sec_label_xattr: Optional[str] = None):
         super().__init__(name)
         assert name == 'CEPH'
+        self.cmount_path = cmount_path if cmount_path else "/"
```

Comment:
Since cmount_path is a string we can simply default to it in the args like `cmount_path: Optional[str] = "/"`. Then explicitly passing a None or empty string is a way the caller can indicate that cmount_path could *not* be used for this item.